packages/plugin-apex/package.json

```diff
@@ -9,8 +9,13 @@
     "@oclif/config": "^1",
     "@oclif/errors": "^1",
     "@salesforce/apex-node": "0.2.8",
+<<<<<<< HEAD
     "@salesforce/command": "3.1.0",
     "@salesforce/core": "2.25.1",
+=======
+    "@salesforce/command": "3.1.3",
```

Comment:
let's use the part below for conflict resolution

Reply:
Good catch! I resolved them locally, but forgot to save and push. Committing now!